megamek/src/megamek/client/ratgenerator/RATGenerator.java

```diff
@@ -1045,7 +1044,7 @@ private void parseChassisNode(int era, Node wn) {
     }
     
     private void parseModelNode(int era, ChassisRecord cr, Node wn) {
-        String modelKey = (cr.getChassis() + " " + wn.getAttributes().getNamedItem("name").getTextContent()).trim();
+        String modelKey = (cr.getChassis() + ' ' + wn.getAttributes().getNamedItem("name").getTextContent()).trim();
```

Comment:
Switching these out is very unlikely to impact performance (since we are dealing with an activity that involves reading and writing data from physical storage) and only contributes negatively to the signal:noise ratio.